ansible_collections/arista/avd/roles/eos_designs/defaults/main/main.yml

```diff
@@ -45,8 +45,8 @@ terminattr_ingestexclude: "/Sysdb/cell/1/agent,/Sysdb/cell/2/agent"
 
 p2p_uplinks_mtu: 9000
 
-underlay_routing_protocol: BGP
-overlay_routing_protocol: BGP
+underlay_routing_protocol: ebgp
```

Comment:
suggested change as per discussion with @ClausHolbechArista 
Move the default *_routing_protocol under node_type_key.
I am updating this PR so it can pass.